docs/snippets/angular/button-story-with-blue-args.ts.mdx

```diff
@@ -0,0 +1,18 @@
+```ts
+// Button.stories.ts
+
+import Button from './button.component';
+export default {
+  title: 'Button',
+  component: Button,
+  parameters: {
+    backgrounds: {
+      values: [
+         { name: 'red', value: '#f00', },
+         { name: 'green', value: '#0f0', },
+         { name: 'blue', value: '#00f', },
+      ]
+    }
+  }
+}
```

Comment:
Apart from the `./button.component` import (rather than `./Button`), this is identical to the react snippet, correct?

Do you think we can just change the name of the import and reuse the snippets--or do you think that'd be too weird for angular folk @jonniebigodes?

I just feel like the less snippets the better for maintainability.

Reply:
Factoring this particular case, angular has it's own ways of naming components, which will then imported into storybook. React works in a different way, ember and vue have their own ways as well. 

Reasoning behind this was the following:
- React (js), we're good in that department, imports are ok.
- React (ts), some additional items are added to the story file, and if we're being accurate then i went with the "carbon copy" (pardon the bad pun), to safe guard us from issues being raised, mentioning that the snippets available don't match the documentation.
- Angular once again was based on the actual implementation as the way the stories are defined when Storybook is added to a project/app.

With this in mind and factoring in your feedback, as this is a pattern that is starting to appear in the documentation, i'm ok with having one platform agnostic snippet, that focus on the information, leaving out any component import references. 

Should any issues be created regarding this case in the repo. I'm more than glad to respond to them.

Let me know if you're ok with it and i'll adjust accordingly.

Reply:
I think that sounds OK, at least to start with. Let's see what folks say when they see the docs for real.